java-client/src/main/java/org/opensearch/client/opensearch/core/search/TermSuggest.java

```diff
@@ -0,0 +1,102 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.client.opensearch.core.search;
+
+import java.util.List;
+import java.util.function.Function;
+
+import org.opensearch.client.json.JsonpDeserializable;
+import org.opensearch.client.json.JsonpDeserializer;
+import org.opensearch.client.json.JsonpMapper;
+import org.opensearch.client.json.ObjectBuilderDeserializer;
+import org.opensearch.client.json.ObjectDeserializer;
+import org.opensearch.client.opensearch.core.search.Suggest.Kind;
+import org.opensearch.client.util.ApiTypeHelper;
+import org.opensearch.client.util.ObjectBuilder;
+
+import jakarta.json.stream.JsonGenerator;
+
+@JsonpDeserializable
+public class TermSuggest extends SuggestBase implements SuggestVariant {
```

Comment:
Should this be SuggestTerm?

Reply:
I'm using the naming conventions from OpenSearch which uses `CompletionSuggest`, `TermSuggest`, and `PhraseSuggest`.